<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cee</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="MiniConsole" name="console">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="122">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="122">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>cee</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionDecimal_dec"/>
    <addaction name="actionEngineering_eng"/>
    <addaction name="actionSI_Units_siu"/>
    <addaction name="actionBinary_bin"/>
    <addaction name="actionHex_hex"/>
    <addaction name="actionOcatal_oct"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentaion"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionHex_hex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hexadecimal (:hex)</string>
   </property>
  </action>
  <action name="actionBinary_bin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Binary (:bin)</string>
   </property>
  </action>
  <action name="actionDecimal_dec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal (:dec)</string>
   </property>
  </action>
  <action name="actionEngineering_eng">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Engineering (:eng)</string>
   </property>
  </action>
  <action name="actionSI_Units_siu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SI Units (:siu)</string>
   </property>
  </action>
  <action name="actionOcatal_oct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ocatal (:oct)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocumentaion">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MiniConsole</class>
   <extends>QPlainTextEdit</extends>
   <header>miniconsole.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
